### DISCUSSION TITLE
Support for remapped backspace key

### DISCUSSION THREAD

**Discussion by AronGreen** [espanso/espanso#1223] [Feature requests and Ideas]
Remapping the backspace key to another key makes espanso not delete the trigger word when expanding.

Example of unexpected/undesired behavior:   
`:date` expands to `:date07/16/2021`

The issue presents itself in Windows with backspace remapped to another key using PowerToys.

[Relevant reddit thread](https://old.reddit.com/r/espanso/comments/olnuui/trigger_word_not_deleted_with_remapped_backspace/?ref=share&ref_source=link)

**Comment by federico-terzi** [comment]
Hey @AronGreen,

Thank you for opening the feature request! We'll investigate :)

Cheers :)

**Comment by directentis1** [comment]
@federico-terzi 
Hi, I recently got the same problem on my Ubuntu (22.04) for unknown reasons.
How can I debug it?

Like as:
:espanshi there1
:tim20;51
:dat05/04/2024

It's worked normally yesterday but maybe something wrong lately.

[Peek 2024-05-04.webm](https://github.com/espanso/espanso/assets/128832114/08360e10-b063-4cd5-ba81-70c2afbd2149)
